contributing/code/core_team.rst

```diff
@@ -170,11 +153,10 @@ A pull request **can be merged** if:
 
 * It is a minor change [1]_;
 
-* Enough time was given for peer reviews (at least 2 days for "regular"
-  pull requests, and 4 days for pull requests with "a significant impact");
+* Enough time was given for peer reviews;
 
-* At least the component's **Merger** or two other Core members voted ``+1``
-  and no Core member voted ``-1``.
+* At least two **Merger Team** members voted ``+1``
```

Comment:
I would really clarify that one +1 vote is enough if the PR author is a Core Member. Otherwise it would be blocking us too often and not reflect reality again.

Reply:
Done